.github/workflows/shellcheck.yml

```
@@ -0,0 +1,23 @@
+name: Shellcheck
+
+# shellcheck gives warnings and suggestions for bash/sh shell scripts.
+# https://github.com/koalaman/shellcheck
+
+permissions:
+  contents: read
+
+on: [workflow_dispatch, pull_request, push]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    name: Shellcheck - Lint
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out ockam repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+      - name: Run ShellCheck
+        run: shellcheck .
```

We should run shellcheck on shell scripts across all subdirectories
```suggestion
        run: shellcheck **/*.sh
```

---

Good point. I wasn't sure if `.` will just work for the whole folder and subfolders

---

Ahh... Does not seem to work though. Should the globbing be `**/**.sh` or maybe `.` just works? 🤔
I haven't really tried it locally